From 64fa15a7ca9c276d28413567c74a1d360fbdc975 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Wed, 31 Jul 2013 23:12:17 +0200
Subject: [PATCH 2/5] Add /usr/share/keymaps to localectl supported locations.

This is the standard upstream location where kbd installs keymaps.
---
 src/locale/localectl.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/locale/localectl.c b/src/locale/localectl.c
index cd7356a..8259c0a 100644
--- a/src/locale/localectl.c
+++ b/src/locale/localectl.c
@@ -538,6 +538,7 @@ static int list_vconsole_keymaps(DBusConnection *bus, char **args, unsigned n) {
         if (!keymaps)
                 return log_oom();
 
+        nftw("/usr/share/keymaps/", nftw_cb, 20, FTW_MOUNT|FTW_PHYS);
         nftw("/usr/share/kbd/keymaps/", nftw_cb, 20, FTW_MOUNT|FTW_PHYS);
         nftw("/usr/lib/kbd/keymaps/", nftw_cb, 20, FTW_MOUNT|FTW_PHYS);
         nftw("/lib/kbd/keymaps/", nftw_cb, 20, FTW_MOUNT|FTW_PHYS);
-- 
1.8.3.2

